lib/nfsc.js

```diff
@@ -696,6 +696,25 @@ class V3 {
     symlink(dir, name, attrs, path, callback) {
         this.client.symlink3(dir, name, attrs, path, callback);
     }
+
+    /**
+      * Procedure READLINK reads the data associated with a
+      * symbolic link.  The data is an ASCII string that is opaque
+      * to the server.  That is, whether created by the NFS
+      * version 3 protocol software from a client or created
+      * locally on the server, the data in a symbolic link is not
+      * interpreted when created, but is simply stored.
+      *
+      * @param {object} object
+      * @param {function} callback(err: null,
+      *                            path: string,
+      *                            obj_attrs: object);
+      * @param {function} callback(err: string,
```

Comment:
in case of error, the NFS server provides the client with additional information